### PR TITLE
Update behavior script to move the arms for cob4-7

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

**Comment by ipa-fxm** [comment]
can you combine this with #411 and either pull them in or close?
related to https://github.com/ipa320/cob_robots/issues/387

**Review by fmessmer — CHANGES_REQUESTED** [review]
can you combine this with #411 and either pull them in or close?
related to https://github.com/ipa320/cob_robots/issues/387

**Comment by NadiaHG** [comment]
why?  they are two different things.... and different files. This pull request implements the **torso_front_cb behavior** for cob4-7, it is already tested on the robot and works. 

The https://github.com/ipa320/cob_robots/pull/411/files use handles for the **pick behavior** and it is still not tested.

**Comment by fmessmer** [comment]
Well, IMHO it is related and I am trying to close old PRs and Issues...so if we test pick on cob4-7, we could also add it to cob4-2...it's all about keeping similar robots similar

**Review by floweisshardt — APPROVED** [review]
tested on cob4-7: OK

**Comment by floweisshardt** [comment]
merging this one as #411 still has issues.

File: cob_default_robot_behavior/scripts/trigger_srvs_cob4-7.py [hunk 1]
```diff
@@ -67,13 +67,28 @@
...
 def torso_front_cb(req):
-    sss.move_base_rel("base",[0,0,1.57],False)
-    sss.move("torso","front",True)
+    handle_arm_left = sss.move("arm_left","side", False)
+    handle_arm_right = sss.move("arm_right","side", False)
+    handle_arm_left.wait()
+    handle_arm_right.wait()
+    if handle_arm_left.get_error_code() == 0 and handle_arm_right.get_error_code() == 0:
+        sss.move_base_rel("base",[0,0,1.57],False)
+        sss.move("torso","front_down_full",True)
```
Comment (fmessmer) [inline cob_default_robot_behavior/scripts/trigger_srvs_cob4-7.py:76]: Well, then this condition is not safe, too, as we do not check whether `sss.move("torso", ...)` was correct....we simply `return TriggerResponse(True, "")`

**Pull request merged**